test/test_infectionsrtfeedback.py

```diff
@@ -39,32 +40,53 @@ def _infection_w_feedback_alt(
     tuple
     """
 
-    Rt = np.array(Rt)  # coerce from jax to use numpy-like operations
     T = len(Rt)
+    Rt = np.array(Rt).reshape(
+        T, -1
+    )  # coerce from jax to use numpy-like operations
     len_gen = len(gen_int)
-    I_vec = np.concatenate([I0, np.zeros(T)])
-    Rt_adj = np.zeros(T)
-
-    for t in range(T):
-        Rt_adj[t] = Rt[t] * np.exp(
-            inf_feedback_strength[t]
-            * np.dot(I_vec[t : t + len_gen], np.flip(inf_feedback_pmf))
-        )
-
-        I_vec[t + len_gen] = Rt_adj[t] * np.dot(
-            I_vec[t : t + len_gen], np.flip(gen_int)
-        )
-
-    return {"post_initialization_infections": I_vec[I0.size :], "rt": Rt_adj}
-
-
-def test_infectionsrtfeedback():
+    I_vec = np.concatenate([I0.reshape(T, -1), np.zeros(Rt.shape)])
+    Rt_adj = np.zeros(Rt.shape)
+    inf_feedback_strength = np.array(inf_feedback_strength).reshape(T, -1)
+
+    for n in range(Rt.shape[1]):
+        for t in range(Rt.shape[0]):
```

Comment:
Prefer some functional programming or list comprehension type thing rather than initializing a 0 `Rt_adj` and filling it in later.

Reply:
I tried using list comprehension rather than initialization and filling in in https://github.com/CDCgov/PyRenew/pull/440/commits/81970595f222ad498753f9974e68b649bd39b3c2  but it doesn't seem to work and the fix is not obvious to me.
Again since this is just for testing, maybe gain in efficiency is not a priority here?